doc/api/process.md

```diff
@@ -472,8 +472,8 @@ added: 6.4.0
 
 * {string}
 
-The `process.argv0` property stores a read-only copy of the original value of
-`argv[0]` passed when Node.js starts.
+`process.argv0` is a configurable property that stores a read-only copy of the
+original value of `argv[0]` passed when Node.js starts.
```

Comment:
Perhaps:

```js
The configurable `process.argv0` property stores the read-only original value of `argv[0]`.
As a configurable property, the value may be replaced using the `delete` and
`Object.defineProperty()` APIs:
```

Then include a quick example?

The reasoning here is that not everyone is going to know automatically what `configurable` and `read-only` actually mean.

Not documenting it is also a valid possibility.